include/bluetooth/services/fast_pair.h

```diff
@@ -56,14 +127,12 @@ size_t bt_fast_pair_adv_data_size(enum bt_fast_pair_adv_mode fp_adv_mode);
  * @param[out] adv_data		Pointer to the Bluetooth advertising data structure to be filled.
  * @param[out] buf		Pointer to the buffer used to store Fast Pair advertising data.
  * @param[in]  buf_size		Size of the buffer used to store Fast Pair advertising data.
- * @param[in]  fp_adv_mode	Fast Pair advertising mode. According to Fast Pair specification,
- *				when no Account Key has been saved on the device both Fast Pair not
- *				discoverable advertising modes result in the same advertising data.
+ * @param[in]  fp_adv_config	Fast Pair advertising config.
  *
  * @return 0 if the operation was successful. Otherwise, a (negative) error code is returned.
  */
 int bt_fast_pair_adv_data_fill(struct bt_data *adv_data, uint8_t *buf, size_t buf_size,
-			       enum bt_fast_pair_adv_mode fp_adv_mode);
+			       struct bt_fast_pair_adv_config fp_adv_config);
```

Comment:
It seems that I guessed correctly :)

https://github.com/nrfconnect/sdk-nrf/pull/7653#discussion_r887017495